plugins/Files/js/constants/files.js

```diff
@@ -42,6 +41,8 @@ export const RENAME_FILE = 'RENAME_FILE'
 export const SHOW_RENAME_DIALOG = 'SHOW_RENAME_DIALOG'
 export const HIDE_RENAME_DIALOG = 'HIDE_RENAME_DIALOG'
 export const SELECT_FILE = 'SELECT_FILE'
+export const SELECT_UP_TO = 'SELECT_UP_TO'
 export const DESELECT_ALL = 'DESELECT_ALL'
 export const DESELECT_FILE = 'DESELECT_FILE'
 export const CLEAR_DOWNLOADS = 'CLEAR_DOWNLOADS'
+
```

Comment:
Go style is to always have a newline at the end of the file, so I lean slightly towards that.